src/libraries/System.Private.CoreLib/src/System/Guid.cs

```diff
@@ -1235,6 +1284,30 @@ public bool TryFormat(Span<char> destination, out int charsWritten, [StringSynta
             return true;
         }
 
+        // Non-vectorized fallback for D, N, P and B formats
+        [MethodImpl(MethodImplOptions.NoInlining)]
```

Comment:
Why NoInlining?  I would have guessed this is unlikely to be inlined, but even if it were, why would we not want that?  This whole path shouldn't exist in general, and when it does, it's because the vectorized path was trimmed away and we're back to where we are today.

Reply:
No idea why I did that, reverted 🙂